src/amo/components/Addon/index.js

```diff
@@ -62,6 +64,7 @@ import './styles.scss';
 
 export class AddonBase extends React.Component {
   static propTypes = {
+    _config: PropTypes.object,
```

Comment:
We usually don't name props with an underscore. I guess it is because it conflicts with the `config` module. I'd personally prefer to import this config module as `defaultConfig` and use `config` as a prop.

Reply:
This is a pattern we have used quite a bit, using an underscore in a prop name to indicate we're using it to inject a dependency. We could do what you suggest, renaming the module on import, but then for consistency we should think about changing it elsewhere, and there are a lot of elsewheres.

Reply:
I do remember we have that for some non-component files but I thought react components did not have such props with underscore, sorry for the noise then.